sdktests/server_side_persistence_base.go

```diff
@@ -14,52 +14,62 @@ import (
 )
 
 func doServerSidePersistentTests(t *ldtest.T) {
-	rdb := redis.NewClient(&redis.Options{
-		Addr:     "localhost:6379",
-		Password: "", // no password set
-		DB:       0,  // use default DB
-	})
-
-	newServerSidePersistentTests(t, &RedisPersistentStore{redis: rdb}).Run(t)
+	if t.Capabilities().Has(servicedef.CapabilityPersistentDataStoreRedis) {
+		rdb := redis.NewClient(&redis.Options{
+			Addr:     "localhost:6379",
+			Password: "", // no password set
+			DB:       0,  // use default DB
+		})
+
+		t.Run("redis", newServerSidePersistentTests(t, &RedisPersistentStore{redis: rdb}).Run)
+	}
 }
 
 type PersistentStore interface {
 	DSN() string
 
+	// TODO: Change these names to something less terrible, or make them even
+	// more generic.
```

Comment:
Since our stores are k/v, perhaps just `Get` and `Set` would be the most familiar names.